types/redux-actions/index.d.ts

```diff
@@ -22,6 +24,12 @@ export interface ActionMeta<Payload, Meta> extends Action<Payload> {
     meta: Meta;
 }
 
+// https://github.com/redux-utilities/redux-actions/blob/v2.3.0/src/combineActions.js#L27
+export interface CombinedActionType {
+    // NOTE: Done on purpose to avoid empty interface declaration
```

Comment:
This will still match basically everything though -- `false` would match for example.

Reply:
Very true, on one hand that's useless, on other hand, returned object is not a `string` after all

Reply:
Super-open for suggestions, that is

Reply:
If this interface is meant to be created by the library and not by the user, you could add a `_combinedActionType: void` field to make this a pseudo-nominal interface.

Reply:
Like this?
```
export interface CombinedActionType {
    _dummy: undefined;
    toString(): string;
}
```

Reply:
Yeah -- at that point `toString` is unnecessary since it's inherited from `Object` anyway:
```ts
interface I {}
function f(i: I): string { return i.toString(); } // No error
```